railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add `bin/update` script to update application automatically
```

Comment:
I think we need to be a bit more descriptive here. What does "update application automatically" really mean? Is that different from deploy? Just to make intents clear.